torch/csrc/jit/tracer.cpp

```diff
@@ -444,6 +444,15 @@ void addInputs(Node* n, const char* name /* unused */, const c10::optional<bool>
 void addInputs(Node* n, const char* name, double value) {
   detail::genericAddInput(n, value);
 }
+void addInputs(Node* n, const char* name /* unused */, const c10::optional<double>& value) {
```

Comment:
Shouldn't `name` be in the comment so that the linter isn't triggered?

Reply:
I just copied `void addInputs(Node* n, const char* name /* unused */, const c10::optional<bool>& value) {` above.